src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/AssumeValidRuleTest.cs

```diff
@@ -97,7 +97,7 @@ public void RunAsync_DoNotSkipValidation_BlockAssumedValidSetOnConsensus_BlockHi
         {            
             this.consensusSettings.BlockAssumedValid = this.concurrentChain.GetBlock(3).HashBlock;
             this.ruleContext.SkipValidation = false;
-            this.ruleContext.ValidationContext.ChainedHeader = this.concurrentChain.GetBlock(4);
+            this.ruleContext.ValidationContext.ChainTipToExtand = this.concurrentChain.GetBlock(4);
```

Comment:
Extend*

Reply:
Yep, that's a typo. Won't fix now because Dan wanted to rename it to smth else because it's not supposed to be CT tip and we shouldn't pass it there from CM.